main.js

```diff
@@ -0,0 +1,15 @@
+
+let displayScreen = document.querySelector('#display')
+console.log(displayScreen)
+
+let buttonClear = document.querySelector('#clear')
+console.log(buttonClear)
+
+let buttonNumbers = document.querySelectorAll(".buttonNum")
+console.log(buttonNumbers)
+
+let buttonOperators = document.querySelectorAll(".buttonOp")
+console.log(buttonOperators)
```

Comment:
Consider how you want to evaluate the expressions once you have them. If you use eval(), as suggested in the README, you might not need to differentiate between numbers and operators.